src/components/GooglePlaceAutocomplete.tsx

```
@@ -135,37 +138,50 @@ const GooglePlaceAutocomplete: FC<GooglePlaceAutocompleteProps> = ({
-          <button
-            type="button"
-            onClick={() => {
-              autoCompleteRef.current!.value = '';
-              if (onClear) onClear();
-            }}
-            className="inset-y-0 right-0 flex items-center pr-3 text-gray-400"
-          >
-            Clear
-          </button>
-        </div>
+        {value && value.toString().length && showClear && (
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)